File: java/src/main/java/mx/iteso/adapter/BackgroundImg.java
package mx.iteso.adapter;
/**
 * Class BackgroundImg.
 */
public class BackgroundImg {
    /**
     * image.
     */
    private Img image;
    /**
     * @param img image.
     */
    public BackgroundImg(final Img img) {
        this.image = img;
    }
    /**
     * @return url.
     */
    public final Img getImage() {
        return this.image;
    }
}

File: csharp/adapter-practice/BackgroundImg.cs


namespace adapter_practice
{
    public class BackgroundImg
    {
        private readonly Img _image;
       
        public BackgroundImg(Img img)
        {
            this._image = img;
        }
       
       public Img GetImage()
        {
            return this._image;
        }
    }
}
